lib/metric.js

```diff
@@ -0,0 +1,26 @@
+function Metric() {
+  this.count = 0;
+  this.time = 0;
+  this.startTime = undefined;
+}
+
+Metric.prototype.start = function() {
+  this.startTime = process.hrtime();
```

Comment:
Every call to `Start` is going to overwrite the previous startTime, but it looks as though we are intending multiple calls to start. This has confusing intent.